scripts/refresh-after-snapshot.py

```
@@ -0,0 +1,644 @@
+    )
+
+    token_resp = run(
+        ["curl", "-sk", f"{kc_url}/realms/master/protocol/openid-connect/token",
+         "-d", "client_id=admin-cli", "-d", "username=admin",
+         "-d", "password=admin", "-d", "grant_type=password"],
+        capture=True,
+    )
+    token_data = json.loads(token_resp.stdout)
+    kc_token = token_data.get("access_token", "")
+    if not kc_token:
+        raise RuntimeError(f"Could not get Keycloak admin token: {token_resp.stdout}")
```

Ruff (0.15.17)

[warning] Avoid specifying long messages outside the exception class

(TRY003)